src/services/rfqm_service.ts

```diff
@@ -471,6 +491,70 @@ export class RfqmService {
         return this._blockchainUtils.isWorkerReadyAsync(workerAddress, gasPrice);
     }
 
+    public async getOrderStatusAsync(orderHash: string): Promise<StatusResponse | null> {
+        const transformSubmission = (submission: RfqmTransactionSubmissionEntity) => {
+            const { transactionHash: hash, createdAt } = submission;
```

Comment:
we stored which tx got mined, might be helpful to add an indication to the transaction as to whether we know it's mined

Reply:
ah I see for successful jobs we only return 1--the one that succeeded

Reply:
yah